recipe/meta.yaml

```diff
@@ -12,9 +12,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   skip: true  # [python_impl == 'pypy']
-  skip: true  # [py>39]
+  skip: true  # [py>310]
```

Comment:
This integer "310" for the version is the only thing that looked fishy to me initially. But the notes on https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#preprocessing-selectors and https://docs.conda.io/projects/conda-build/en/stable/user-guide/environment-variables.html#build-envs _imply_ that "py>310" is the way to write it, even though it's not explicitly said for 310.

Reply:
Yeah I checked with the openmm feedstock too and they seem to have successfully used "310". That being said looking at a few "python 3.10 update" PRs in other feedstocks, it's not immediately clear to me that the py3.10 runners actually get triggered most of the time.